src/test/rgw/test_rgw_iam_policy.cc

```diff
@@ -76,7 +76,10 @@ using rgw::IAM::s3ListMultipartUploadParts;
 using rgw::IAM::None;
 using rgw::IAM::s3PutBucketAcl;
 using rgw::IAM::s3PutBucketPolicy;
-using rgw::Service;
```

Comment:
this is a conflict issue with recent backport (https://github.com/ceph/ceph/pull/30579). please change to: ``rgw::Service``